katello/katello-change-hostname

```diff
@@ -237,6 +238,12 @@ def hammer_cmd(cmd, exit_codes=[0], message=nil)
   run_cmd("hammer -u #{@options[:username].shellescape} -p #{@options[:password].shellescape} #{cmd}", exit_codes, message)
 end
 
+def get_default_proxy_id
+  output = hammer_cmd("--output json capsule list --name #{@old_hostname}")
+  proxy = JSON.parse(output)
+  proxy.any? ? proxy.first["Id"] : false
```

Comment:
that won't work anymore. `info` returns a single proxy (if any), so `.any?` and `.first` are not needed.

Reply:
this also means that you'd never return `false` as `hammer_cmd` would error out directly, when the old capsule is not found:

```
[root@sat-r220-02 ~]# hammer --output json capsule info --name sat-r220-02.lab.eng.rdu2.redhat.comee
Error: smart_proxy not found
[root@sat-r220-02 ~]# echo $?
70
```